data/human/news.txt

```diff
@@ -344,6 +421,7 @@ news "geological engineer"
 			"I have equipment that would let me trigger an earthquake just by pressing a button. That's why they never allow me into the office when I'm drunk."
 			"For centuries people have been saying that asteroid mining is going to put terrestrial resource extraction companies out of business, but it still hasn't happened."
 			"A few times, I've traveled so deep underground that the rock felt warm to the touch. But I imagine that's nowhere near as exciting as traveling through space."
+			"I can't believe how annoying it must've been in the distant past to be a geological engineer, with you on one side, the thing you're trying to study on the other, and 15 kilometres of solid rock inbetween. Thankfully, technology and scanners have marched on since then, and those depressing days are long gone."
```

Comment:
```suggestion
			"I can't believe how annoying it must've been in the distant past to be a geological engineer, with you on one side, the thing you're trying to study on the other, and 15 kilometres of solid rock in between. Thankfully, technology and scanners have marched on since then, and those depressing days are long gone."
```